libexpat/libexpat
# Link against libm for function "isnan" (fixes #509)

Fixes #509

To test with GNU Autotools, locally:

```bash
cd "$(mktemp -d)"
git clone --branch issue-509-link-against-libm --depth 1 https://github.com/libexpat/libexpat
cd libexpat/expat/
./buildconf.sh && ./configure && make all check
```

To test with CMake, locally:

```bash
cd "$(mktemp -d)"
git clone --branch issue-509-link-against-libm --depth 1 https://github.com/libexpat/libexpat
cd libexpat/expat/
cmake . && make VERBOSE=1 all test
```
